terra/compute/container.py

```diff
@@ -37,6 +37,10 @@ def pre_run(self):
     # for special executors, etc...
     super().pre_run()
 
+    if settings.logging.server.family == 'AF_UNIX':
+      self.add_file_readonly(settings.logging.server.listen_address,
+                             settings.logging.server.listen_address)
```

Comment:
It seems unusual to mount the file socket to the exact same location within the container.  Is this because the `volume_map` does not apply to this setting?